vignettes/fims-demo.Rmd

```diff
@@ -19,12 +19,12 @@ The NOAA Fisheries Integrated Modeling System (FIMS) is a new modeling framework
 ## Creating Models in FIMS
 To begin, we import the FIMS and TMB libraries. We then create a new Rcpp fims module using `Rcpp::Module` and call `Fims$clear` to ensure C++ memory from any previous fims model run is cleared out. 
```

Comment:
This description should be updated to describe the new loadModule and clear() function are used.